Mathlib/Init/Data/List/Default.lean

```diff
@@ -0,0 +1,10 @@
+/-
```

Comment:
What is the purpose of having this file?

Reply:
To remove an entry in the port dashboard.